python/core/auto_generated/qgsvectordataprovider.sip.in

```diff
@@ -216,6 +217,7 @@ but subclasses can override this method to handoff calculation of aggregates to
 :param parameters: parameters controlling aggregate calculation
 :param context: expression context for filter
 :param ok: will be set to ``True`` if calculation was successfully performed by the data provider
+:param fids: Pointer to a list of feature ID to filter out unwanted features in the request
```

Comment:
"Pointer to a list of feature IDs to use in the calculation. Set to NULLPTR to disable any ID based filtering. An empty list indicates that no features will be considered."

(Make sure NULLPTR is all capitals, it's a special keyword)

Reply:
Done